ansible/wordpress.yml

```diff
@@ -33,6 +35,17 @@
         src: config.php
         # https://superuser.com/a/559371/102684
         dest: /etc/wordpress/config-default.php
+        owner: "{{ apache_user }}"
+        group: "{{ apache_group }}"
+
+    # https://help.ubuntu.com/community/WordPress#Install_WordPress
+    - name: Fix permissions on WordPress directory
+      file:
+        path: "{{ wp_install_dir }}"
+        owner: "{{ apache_user }}"
+        group: "{{ apache_group }}"
+        state: directory
+        recurse: true
```

Comment:
Not positive that we _want_ to allow automatic updates...? That would mean WordPress itself wouldn't be deployed immutably. Happy to take the conversation to an issue.

Reply:
If the upstream source that deploys Wordpress on the instance is in sync with the updates, it may not be too bad. Wordpress updates download into /wp-content/updates and then deploy into the main directory. Sometimes those updates contain schema updates for the database as well. Lots to consider.

Reply:
So, the way that I did this for our modified version of cf-ex-wordpress (deploying in cloud.gov) was to do some json in a particular schema, and feed that to wp-cli. 

Relevant python bits here: https://github.com/18F/cf-ex-wordpress/pull/1/commits/25be580fd34942c5314a22a2cfa67a463bc6e39f#diff-81085e34fc3ba4cf38cba76d477219bdR59 . I'm sure that could be turned into some ansible code.

Reply:
Let's worry about it separately, if that's ok with you both: #23.